docs/explainers.md

```diff
@@ -0,0 +1,152 @@
+# Model Interpretability on Spark
+
+## Model Interpretability
+
+Model interpretability helps developers, data scientists and business stakeholders in the organization gain a comprehensive understanding of their machine learning models. It can also be used to debug models, explain predictions and enable auditing to meet compliance with regulatory requirements.
+
+## Why run model interpretation on Spark
+
+Model Interpretability on Spark enables users to interpret a black box model at massive scales with the Apache Spark™ distributed computing ecosystem. Various components support local interpretation for tabular, vector, image and text classification models, with two popular model-agnostic interpretation methods: Local Surrogate (LIME) and Kernel SHAP.
+
+## Usage
+
+To see examples of model interpretability on Spark in action, take a look at these sample notebooks:
+
+- [Tabular SHAP explainer](../notebooks/Interpretability%20-%20Tabular%20SHAP%20explainer.ipynb)
+- [Image explainers](../notebooks/Interpretability%20-%20Image%20Explainers.ipynb)
+- [Text explainers](../notebooks/Interpretability%20-%20Text%20Explainers.ipynb)
+
+|                        | Tabular models              | Vector models             | Image models            | Text models           |
+|------------------------|-----------------------------|---------------------------|-------------------------|-----------------------|
+| LIME explainers        | [TabularLIME](#TabularLIME) | [VectorLIME](#VectorLIME) | [ImageLIME](#ImageLIME) | [TextLIME](#TextLIME) |
+| Kernel SHAP explainers | [TabularSHAP](#TabularSHAP) | [VectorSHAP](#VectorSHAP) | [ImageSHAP](#ImageSHAP) | [TextSHAP](#TextSHAP) |
+
+### Common local explainer params
```

Comment:
Perhaps a breif explanation of what a local explainer is might make this feel less abrupt and help people understand that all current explainers have these methods

Reply:
Done.